src/components/Filters.tsx

```diff
@@ -39,52 +69,108 @@ const Filters: React.FC<FiltersProps> = ({
   handleParatimesChange,
   handleMaintainedByOasisToggle,
   handleClearTags,
-  handleClearLangs,
 }) => {
+  const handleTagsChange = (event: SelectChangeEvent<typeof selectedTags>) => {
+    const {
+      target: { value },
+    } = event;
+    const newTags = typeof value === 'string' ? value.split(',') : value;
+    newTags.forEach((tag: string) => handleTagClick(tag));
+  };
+
+  const sortedTags = [...tags].sort();
+
+  const sortedLicenses = [...licenses].sort((a, b) => {
+    if (a === '') return 1;
+    if (b === '') return -1;
```

Comment:
What do these do

Reply:
This places an item with an empty string ('Unspecified' license) to be the last one in order.

Reply:
I'd add a comment